tests/gis_tests/backends/postgis/tests.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+
+from django.db import connection
+from django.test import TestCase
+
+
+@unittest.skipUnless(connection.vendor == "postgresql", "PostgreSQL specific tests")
+class TestsPostGISCreateExtension(TestCase):
+    def _extension_exists(self):
+        with connection.cursor() as cursor:
+            cursor.execute("SELECT 1 FROM pg_extension WHERE extname = 'postgis'")
+            return cursor.fetchone()[0] is not None
+
+    def test_extension_is_created(self):
+        """Create PostGIS extension if it does not exist."""
+        # Make sure extension is not there before we test creation
+        with connection.cursor() as cursor:
+            cursor.execute("DROP EXTENSION IF EXISTS postgis CASCADE")
```

Comment:
This can cause as isolation issues :thinking: Give me few minutes to figure out if it's worth testing.

Reply:
Testing `prepare_database()` is difficult (a lot of mocking is needed to get a proper isolation). IMO we can push this without tests.

Reply:
If you feel fine having no tests here, sure thing. The change is small and if we don't have yet good mocking structure for ORM-database operations, then it will take quite some efforts to introduce it.

Reply:
Creating and isolating new connections in tests are tricky.

>  if we don't have yet good mocking structure for ORM-database operations

Testing database operations is not difficult, but here we want to test a low-level operations related with preparing a database before using ORM.